app.js

```diff
@@ -0,0 +1,20 @@
+const express = require("express");
+
+const app = express();
+
+app.use(express.json());
+
+const { getTopics } = require("./controllers/topics.controllers.js");
```

Comment:
To keep this file tidy, I'd consider moving this require further up the the file at the top where the other `require` is.